scram-tools.file/tools/professor2/professor2.xml

```diff
@@ -8,4 +8,6 @@
 <use name="yoda"/>
 <use name="eigen"/>
 <runtime name="PATH" value="$PROFESSOR2_BASE/bin" type="path"/>
+<runtime name="PYTHONPATH" value="$PROFESSOR2_BASE/lib/python3.9/site-packages" type="path"/>
```

Comment:
We avoid having individual packages adding their paths to `PYTHONPATH`, so please remove this line. `professor2` is  already a dependency of https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_14_1_X/master/python_tools.spec so its `$PROFESSOR2_BASE/lib/python3.9/site-packages` will be available at runtime.